.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Should we exclude all these configs from PR?

Reply:
These files are a part of a workflow where users don't have to worry about their formatting and it is expected to be enforced at the repository level. This particular combo usually amounts to it "just working" across devices, where many other setups end up with CRLF issues and some confusing steps. These settings override User-defined VS Code settings, but ones that are not specified here can still be specified at the user or workspace level. We could alternatively add prettier checks at the .github level so that PRs can have failing checks when code hasn't been prettier-ified, but I find that devs get frustrated when they check in code that needs to go back and be formatted.

I did realize I missed adding vscode-prettier to the recommended extensions for this project, so that's something I'd recommend when enforcing prettier.